src/view/ClusterDetailsPanel.ts

```diff
@@ -160,7 +160,7 @@ export class ClusterDetailsPanel {
 
 				if (selectedCluster.length > 0) {
 					managedClusters = await this._kubeDataLoader.loadManagedCluster(selectedCluster);
```

Comment:
Add try catch as done for klusterlet in line 185

Reply:
try/catch removed and replaced by suitable return statement in kube.ts